Lib/argparse.py

```diff
@@ -553,7 +553,10 @@ def _format_action_invocation(self, action):
                 default = self._get_default_metavar_for_optional(action)
                 args_string = self._format_args(action, default)
                 for option_string in action.option_strings:
-                    parts.append('%s %s' % (option_string, args_string))
+                    if args_string:
+                        parts.append('%s %s' % (option_string, args_string))
+                    else:
+                        parts.append('%s' % (option_string))
```

Comment:
I've fixed it.  I haven't worked with the older version of string formatting, so I wasn't aware I did a hybrid of the two methods.  It seems that a 1-element tuple is preferred, so I've changed it to be that.